components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -202,16 +232,234 @@ The member expression dialog also supports tokens, because candidate tokens are
 
-## Removing Legacy Policies
+### Projects
 
-Once you've rewritten your v1 policies as v2 policies, you should remove the v1 legacy policies.
-This section walks you through deleting all of the legacy policies imported into v2, which will provide you a clean slate for exploring the IAM v2 Beta.
+Projects are used to group and permission Chef Automate resources, ingested client-runs, and compliance nodes.
 
-Open the menu on any custom policy, located at the end of the policy row, and select **Delete Policy** from the menu.
-(You will not have this option for Chef-managed policies.)
-A warning appears if members are still attached to the policy, because deleting that policy disrupts access for all of its members.
-However, you can still delete the policy.
-Repeat for each legacy policy.
+Projects can be managed via the Projects list under the *Settings* tab and consist of an ID, a name, and a collection of rules. Project ingest rules are lists of conditions used only when
+[assigning ingested resources to projects]({{< relref "iam-v2-guide#assigning-ingested-resources-to-projects" >}}),
+so they are not relevant when assigning IAM resources such as teams or roles.
+
+#### Creating a Project
+
+To create a project, navigate to the Projects list under the **Settings** tab and select **Create Project**. You will need to provide a name and can optionally edit the ID. You must create a project before you can assign any resources to it.
+
+#### Assigning Resources to Projects
+
+You can use the browser to assign teams and tokens to projects. To assign a team to projects, select a team from the _Teams_ list, then select **Details**.
+Likewise, to assign a token to projects, select a token from the API tokens list, then select **Details**.
+In either case, you can select projects from the projects dropdown to assign.
+
+You may also assign teams and tokens to projects on creation. In the creation modal, select any projects to which the new resource should belong.
+
+Presently, policies and roles can only be assigned to projects using the CLI, not the browser. Users cannot be assigned to projects from the browser or CLI.
+
+If you would like to delegate ownership of a project to another user so that they may assign resources, you will want to make that user a [Project Owner]({{< relref "iam-v2-guide.md#project-owners" >}}) of that project.
+
+#### Assigning Ingested Resources to Projects
+
+While resources created within Automate can be directly assigned to a project, ingested resources, including Compliance reports, events, and nodes, are created outside of Automate and therefore must be assigned to projects a different way.
+
+Project rules are used to associate ingested resources with projects within Automate. A project rule contains conditions that determine if an ingested resource should be moved into the rule's project.
```

Comment:
should we mention here that audit cookbook 7.5+ is required to ensure those fields are populated on compliance reports?

Reply:
oh nevermind, i see that's called out further down!

Reply:
@vjeffrey i was curious do we call out that audit cookbook anywhere else in the automate docs? just curious, cause if so we can link to it somewhere in these docs as well.

Reply:
oh shoot i saw this but forgot to look. i’m afk but i was just gonna search our docs on the site for “audit” if anyone wants to search

Reply:
oh shoot i saw this but forgot to look. i’m afk but i was just gonna search our docs on the site for “audit” if anyone wants to search

Reply:
i can also check myself later tonite!